SH4RE/SH4RE/ListingViewModel.swift

```diff
@@ -612,10 +671,7 @@ func sendRentalStatusMessage(statusMessage: String, messagePreview: String, user
                 return
             }
         }
-    })
-
-    
-    
+    }) 
 }
 
 //returns status code 0:pending, 1:accepted, 2:declined, 3:cancelled, 4:error
```

Comment:
should this stuff become enum too?

Reply:
This wasn't part of this merge but we could make a ticket